frontend/src/app/projects/models/model-source/choose-source.component.html

```diff
@@ -9,12 +9,7 @@ <h2 class="title text-center">Git sources</h2>
     <a (click)="modelSourceSelection.emit('git-add')">
       <mat-card class="option-card">
         <p class="subtitle">Use existing repository</p>
-        <div class="flex justify-center">
-          <img
-            class="git"
-            src="https://git-scm.com/images/logos/downloads/Git-Logo-1788C.png"
-          />
-        </div>
+        <app-mat-icon size="50px">link</app-mat-icon>
```

Comment:
I have the feeling that you used the link icon in order to make it consistent still I do prefer the Git icon - but this is just a personal preference

Reply:
I discussed that with Moritz and we decided to switch to the `Link` for the reason you mentioned. Especially when we do want to also add a "Create repository" for git in the future, consistently using the same icons for "Create" and "Link" enhances usability (i.e., as you can just click on the button with the familiar icon).

Reply:
alright fine by me